From fa5c4b2e60a98944863097b448960d0744916b1f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Wed, 13 Feb 2019 22:32:44 +0100
Subject: [PATCH] nologin: Install applet to /sbin instead of /usr/sbin

This is required to retain compatibility with our old custom nologin
applet written in C which was also installed to /sbin.

Compatibility with the old path is required because login shell paths
are hardcoded in /etc/passwd.
---
 util-linux/nologin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util-linux/nologin.c b/util-linux/nologin.c
index 5e5e42305..b03470ac6 100644
--- a/util-linux/nologin.c
+++ b/util-linux/nologin.c
@@ -19,7 +19,7 @@
 //config:	If you know these will be available externally you can
 //config:	disable this option.
 
-//applet:IF_NOLOGIN(APPLET_SCRIPTED(nologin, scripted, BB_DIR_USR_SBIN, BB_SUID_DROP, nologin))
+//applet:IF_NOLOGIN(APPLET_SCRIPTED(nologin, scripted, BB_DIR_SBIN, BB_SUID_DROP, nologin))
 
 //usage:#define nologin_trivial_usage
 //usage:	""
